caom2utils/caom2utils/fits2caom2.py

```diff
@@ -1543,12 +1549,6 @@ def augment_artifact(self, artifact):
             'Begin artifact augmentation for {} with {} HDUs.'.format(
                 artifact.uri, len(self.headers)))
 
-        if self.blueprint.get_configed_axes_count() == 0:
```

Comment:
I am not aware of FITS with only non-WCS  data. If you know of a case, I will put the check back.

Generally, this check was meant to deal with the case that should now be handled by using the GenericParser class instead of the FitsParser class.

Reply:
But then what is the reason this was removed?

Reply:
Raises an exception.